src/invidious/helpers/i18next.cr

```diff
@@ -189,7 +189,7 @@ module I18next::Plurals
 
     # Emulate the `rule.numbers.size == 2 && rule.numbers[0] == 1` check
     # from original i18next code
-    private def is_simple_plural(form : PluralForms) : Bool
+    private def simple_plural?(form : PluralForms) : Bool
```

Comment:
Can you revert that change, as it's already in #4806 ?

Reply:
Done. Sorry about that